src/components/Button/Button.scss

```diff
@@ -349,11 +349,16 @@
     background-color: transparent;
   }
 
-  .ms-Icon {
+  .ms-Icon--m {
     font-size: $ms-icon-size-m;
     padding: 8px;
   }
 
+  .ms-Icon--s {
```

Comment:
I think we should be removing the usage of these font size classes in components... they aren't semantic and don't really mean anything.